src/MacSrc/Modding.c

```diff
@@ -87,7 +87,7 @@ int ProcessModDirectory(char* dirname) {
 		while(ep = readdir(dp)) {
 
 			printf("ep->d_name %s\n", ep->d_name);
-			char buf[strlen(dirname) + strlen(ep->d_name) + 2];
+			char buf = malloc(strlen(dirname) + strlen(ep->d_name) + 2);
```

Comment:
This'll introduce a memory leak unless it's freed.

Reply:
I figured as much. Again MSVC rejected the non-constant array length.

I'll see if I can add the freeing